sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-student-unmet-need-report.sql

```diff
@@ -0,0 +1,68 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Student_Unmet_Need_Report',
+    'SELECT
+      users.first_name AS "Student First Name",
+      users.last_name AS "Student Last Name",
+      sin_validations.sin AS "SIN",
+      applications.student_number AS "Student Number",
+      users.email AS "Student Email Address",
+      students.contact_info ->> ''phone'' AS "Student Phone Number",
+      institution_locations.institution_code AS "Institution Location Code",
+      institution_locations.name AS "Institution Location Name",
+      applications.application_number AS "Application Number",
+      student_assessments.assessment_date AS "Assessment Date",
+      education_programs_offerings.offering_intensity AS "Study Intensity (PT or FT)",
+      students.disability_status AS "Profile Disability Status",
+      applications.data ->> ''disabilityStatus'' AS "Application Disability Status",
+      education_programs_offerings.study_start_date AS "Study Start Date",
+      education_programs_offerings.study_end_date AS "Study End Date",
+      education_programs.program_name AS "Program Name",
+      education_programs.credential_type AS "Program Credential Type",
+      education_programs.cip_code AS "CIP Code",
+      education_programs.completion_years AS "Program Length",
+      education_programs.sabc_code AS "SABC Program Code",
+      education_programs_offerings.offering_name AS "Offering Name",
+      education_programs_offerings.year_of_study AS "Year of Study",
+      applications.data ->> ''indigenousStatus'' AS "Indigenous person status",
+      applications.data ->> ''citizenship'' AS "Citizenship Status",
+      applications.data ->> ''youthInCare'' AS "Youth in Care Flag",
+      applications.data ->> ''custodyOfChildWelfare'' AS "Youth in Care beyond age 19",
+      applications.data ->> ''relationshipStatus'' AS "Marital Status",
+      applications.data ->> ''dependantstatus'' AS "Independant/Dependant",
+      student_assessments.workflow_data -> ''calculatedData'' ->> ''totalEligibleDependents'' AS "Number of Eligible Dependants Total",
```

Comment:
I believe the correct variable would be `calculatedDataTotalEligibleDependants`. Does it make sense?